src/main/java/io/stargate/sgv2/jsonapi/config/DocumentLimitsConfig.java

```diff
@@ -56,4 +56,13 @@ public interface DocumentLimitsConfig {
   @Positive
   @WithDefault("100")
   int maxArrayLength();
+
+  /**
+   * @return Maximum length of Vector ($vector) array JSON API allows -- NOTE: backend data store
+   *     likely limits length to a lower value; but we want to prevent handling of huge arrays
+   *     before trying to pass them to DB.
+   */
+  @Positive
+  @WithDefault("8000")
```

Comment:
8000 is probably ok given that OpenAI's recent models output vectors of length 1536, and other libs I've worked with generally output vectors of length around 1000-2000.

However, [older OpenAI models would output embeddings of around 12k](https://openai.com/blog/new-and-improved-embedding-model). So would there be anything blocking us from increasing this limit to 20k or even 50k to be safe?

Reply:
@vkarpov15 Just memory usage and decoding overhead, but that should be ok at this range.
How about I just set it as 16000 then for now.